index.html

```diff
@@ -12,7 +12,7 @@
     <header class="navbar">
         <a href="#" class="nav-logo">My Logo</a>
```

Comment:
- [ ] Please fix this margin so your design looks like the design , please align the items in the works section using grid
![image](https://user-images.githubusercontent.com/74744787/185764888-226135c8-daa9-4c8c-ac58-121b9a01ffdc.png)